test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -0,0 +1,163 @@
+Describe "Set/New-Service cmdlet tests" -Tags "Feature","RequireAdminOnWindows" {
+  BeforeAll {
+    $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
```

Comment:
What is indentation in the file? 2 or 4?

Reply:
Seems to be due to VSCode.  Let me check my editor settings.

Reply:
I reformatted the whole file so it should at least be consistent